### ISSUE TITLE
Have all close icons be on the left-hand side

### ISSUE THREAD

**Issue by pgoodman** [trailofbits/qt-multiplier#99]
Right now, close buttons are all on the right-hand side: 
![image](https://user-images.githubusercontent.com/43664/224387207-8aeeb412-9cf1-4d1d-ad08-b1296ce32844.png)
![image](https://user-images.githubusercontent.com/43664/224387267-61ffa5c7-221f-45f3-be53-caf4e13844b5.png)

This makes it hard to delete a lot of tabs at once, because it means moving the mouse around to the right side of the next tab to close. If the buttons were left-aligned, then you could click to delete, and you'd know that the next tab's close button would land underneath your mouse again to enable quick deleting of that tab as well.

TL;DR: I want to minimize mouse movement when multi-deleting a run of tabs.
